spec/helpers/feature_flag_spec.rb

```diff
@@ -0,0 +1,66 @@
+require "rails_helper"
+
+UserStruct = Struct.new(:flipper_id)
+
+describe FeatureFlag, type: :helper do
+  describe ".enabled?" do
+    it "calls Flipper's enabled? method" do
+      allow(Flipper).to receive(:enabled?).with("foo")
+
+      described_class.enabled?("foo")
+
+      expect(Flipper).to have_received(:enabled?).with("foo")
+    end
+  end
+
+  describe ".exist?" do
+    it "calls Flipper's exist? method" do
+      allow(Flipper).to receive(:exist?).with("foo")
+
+      described_class.exist?("foo")
+
+      expect(Flipper).to have_received(:exist?).with("foo")
+    end
+  end
+
+  describe ".accessible?" do
+    let(:user) { UserStruct.new(flipper_id: 1) }
+
+    it "returns false when flag doesn't exist" do
+      expect(described_class.accessible?("missing_flag")).to be_truthy # rubocop:disable Rspec/PredicateMatcher
```

Comment:
No blocker for this PR, but for a future refactor it may be easier to move the `rubocop:disable` to the top and add a `rubocop:enable Rspec/PredicateMatcher` at the bottom of the file as we do in some other files.

https://rubocop.readthedocs.io/en/latest/configuration/#disabling-cops-within-source-code